test/flutter_redux_test.dart

```diff
@@ -538,7 +537,6 @@ void main() {
       final widget = () => StoreProvider(
             store: store,
             child: StoreBuilder<String>(
-              onWillChange: counter,
```

Comment:
Please do not remove this line, rather please adjust it for the new function signature to make the test pass.